src/NetPanzer/Views/Game/VehicleSelectionView.cpp

```diff
@@ -554,7 +554,7 @@ void VehicleSelectionView::drawMiniProductionStatus(Surface &dest)
 {
     char strBuf[256];
     char outpostNameBuf[256];
-    char outpostUserNameBuf[256];
+    std::string outpostUserNameBuf;
```

Comment:
I hope this change won't slow things down too much. I'm not really knowledge about cpp automatic memory management related to strings.

Reply:
I think most of the time it will be stack allocated and less than the old size of 256 bytes anyway.